nimble/controller/src/ble_ll_conn.c

```diff
@@ -3598,8 +3769,10 @@ int
 ble_ll_conn_rx_isr_end(uint8_t *rxbuf, struct ble_mbuf_hdr *rxhdr)
 {
     int rc;
-    int is_ctrl;
+    uint8_t is_ctrl;
+    uint8_t is_data;
```

Comment:
looks like `is_data` is used only when
MYNEWT_VAL(BLE_LL_CFG_FEAT_CTRL_TO_HOST_FLOW_CONTROL  is set.